Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -352,20 +352,25 @@ private void StandAloneThreadExceptionHandler(object sender, UnhandledExceptionE
         public bool OpenSolution(string solutionFolder)
         {                      
             try
-            {               
+            {
+                Console.WriteLine("Folder in WorkSpace" + solutionFolder);
                 Reporter.ToLog(eLogLevel.INFO, string.Format("Loading the Solution '{0}'", solutionFolder));
                 LoadingSolution = true;
 
                 //Cleaning previous Solution load
                 Reporter.ToLog(eLogLevel.DEBUG, "Loading Solution- Cleaning previous Solution items");
+                Console.WriteLine("Before Close Solution");
                 CloseSolution();
+                Console.WriteLine("After Close Solution");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)